clothcalc_placeholder.js

```diff
@@ -5493,9 +5499,17 @@
                             spot,
                             count,
                             job_id,
-                            dat,
-                            tmp,
-                            timestamp;
+                            dat = get_server_date(),
+                            datMonth = dat.getMonth(),
+                            resetHour = datMonth > 2 && datMonth < 10 ? 3 : 2,
```

Comment:
Uh, I'll pretend I didn't see this... :D Too bad Inno doesn't have moment.js already loaded in TW, it would be so easy to do this properly (so that it works in all days of March and October).